pkg/az/group.go

```diff
@@ -20,7 +20,7 @@ type GroupCommand struct {
 	Location    string `yaml:"location"`
 }
 
-func (c *GroupCommand) HandleError(cxt *context.Context, err builder.ExitError, stdout string, stderr string) error {
+func (c *GroupCommand) HandleError(cxt context.Context, err builder.ExitError, stdout string, stderr string) error {
```

Comment:
this is super minor but we probably could change to this so it follows the convention of abbreviation for naming context
```suggestion
func (c *GroupCommand) HandleError(ctx context.Context, err builder.ExitError, stdout string, stderr string) error {
```